libsel4/sel4_plat_include/tqma8xqp1gb/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021, Breakaway Consulting Pty. Ltd.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#ifdef HAVE_AUTOCONF
```

Comment:
please remove the check for `HAVE_AUTOCONF` and include `<autoconf.h>` unconditionally.

Reply:
Fixed.